### PR TITLE
Temporary script for ingesting VoC data [Now ready for merge]

### PR DESCRIPTION
**Pull request by fojackson8**
Please don't actually merge this code, just use for one time ingestion then delete

### CODE REVIEW

**Comment by fojackson8** [comment]
UPDATE: This can now be merged, as we've set a public URL for the voc data

**Review by iamleeg — APPROVED** [review]

File: ingestion/functions/parsing/variants/variants_test.py [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+_SOURCE_URL = "placeholder_URL"
+
+
+class VariantsTest(unittest.TestCase):
+    def setUp(self):
+        self.maxDiff = 5000
+
+    def test_parse(self):
+        '''
+        Includes a row where province and district are unspecified, where it should return just
```
Comment (AnyaLindstromBattle) [inline ingestion/functions/parsing/variants/variants_test.py:15]: What does this mean?

**Review by AnyaLindstromBattle — APPROVED** [review]
Just had one question about a note in variants_test.py  but otherwise looks good.

File: ingestion/functions/parsing/variants/variants_test.py (hunk 1 above)
Comment (fojackson8) [reply ingestion/functions/parsing/variants/variants_test.py]: Good catch was a copying error, now removed

**Pull request merged**